tests/appsec/integrations/django_tests/conftest.py

```diff
@@ -77,3 +77,19 @@ def test_spans(tracer):
         yield container
         _end_iast_context_and_oce()
         container.reset()
+
+
+@pytest.fixture
+def test_spans_iast_disabled(tracer):
```

Comment:
I don't understand why this fixture is called "iast_disabled" and the first thing it does is enabling iast?

Reply:
it's enabled but the sampling rate is 0%, let me update this name